### PR TITLE
Update installation commands

### PR DESCRIPTION
**Pull request by vkucera**
@ddobrigk @aalkin @jgrosseo

### CODE REVIEW

File: docs/gettingstarted/installing.md [hunk 1]
```diff
@@ -88,31 +88,31 @@ alienv enter O2Physics/latest ninja/latest
...
 ```bash
-ninja install PWGCF/Tasks/all
+ninja PWGCF/Tasks/install
 ```
 
 A specific executable can be built with
 
 ```bash
-ninja install O2Physicsexe-<target>
+ninja O2Physicsexe-<target>
```
Comment (jgrosseo) [inline docs/gettingstarted/installing.md:109]: So here it will not be installed anymore?

Comment (jgrosseo) [reply docs/gettingstarted/installing.md:109]: Then we should also explain this. I think from the current instructions, people would expect that the binary is also in the path. So we have to explain here in addition, how to execute it in the stage directory.

Comment (aalkin) [reply docs/gettingstarted/installing.md:109]: Good point. @vkucera could you please add a sentence about that?

Comment (ktf) [reply docs/gettingstarted/installing.md:109]: I think the best would be to have a target which installs the executable called like the executable itself. it should be doable, at least with ninja

Comment (vkucera) [reply docs/gettingstarted/installing.md]: @aalkin @jgrosseo Updated

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**